packages/xo-server/src/xo-mixins/backups-ng/index.js

```diff
@@ -1146,14 +1146,28 @@ export default class BackupNg {
         $defer.call(xapi, 'deleteVm', snapshot)
       }
 
+      const compress = getJobCompression(job)
+      const { $pool: pool } = snapshot
+      if (
+        compress === 'zstd' &&
+        pool.restrictions.restrict_zstd_export !== 'false'
```

Comment:
I'm not sure this is correct, systems on which zstd is unsupported (e.g. XenServer) will not have such an entry.

Reply:
If the system not support it, the entry value will be `undefined`. 
So, it's different than `false` and the warning will be displayed because the `zstd` compression is selected.

Reply:
That looks good then :+1: